lib/rubocop/cop/layout/extra_spacing.rb

```diff
@@ -98,6 +98,8 @@ def should_aligned_with_preceding_line?(token)
         end
 
         def check_other(token1, token2, ast)
+          return false if allow_for_trailing_comments? && token2.text[0] == '#'
```

Comment:
```suggestion
          return false if allow_for_trailing_comments? && token2.text.start_with?('#')
```

Reply:
D'oh, yeah, I forgot that existed.  :-)

Reply:
Ha, yeah, I can look into it for another PR!  :-)

Reply:
Hmmm, current cop Performance/StartWith suggests `start_with?` rather than a regex anchored to the start.  I'm thinking subbing it in for `[0] ==` wouldn't be so much a _performance_ issue as a _style_ issue.  Or is there something unperformant about `[0] ==` that I'm just not seeing?  I could try to do it either way, just wondering whether I could plausibly add it to the extant cop, or I should start a new one.